### DISCUSSION TITLE
Ignore relate schemas when pull data from Sql Server

### DISCUSSION THREAD

**Discussion by tddat15** [prisma/prisma#22083] [Q&A]
### Question

My database is too large (> 4Mg when I pull full database multiSchema previewFeature), so I have tried to reduce it by providing some schema I am really using in  previewFeatures = ["multiSchema"], but it throws the error below.
It still require other relate schemas, what should I do?

✖ Introspecting based on datasource defined in prisma\test_mssql_schema.prisma

Error: P4002

The schema of the introspected database was inconsistent: The schema of the introspected database was inconsistent: Cross schema references are only allowed when the target schema is listed in the schemas property of your datasource. `BILLING.ACCOUNT` points to `PROPERTY.ADDRESS` in constraint `FK_ACCOUNT_ADDRKEY`. Please add `PROPERTY` to your `schemas` property and run this command again.

**Comment by tddat15** [comment]
I already tried to add relationMode = "prisma" into the file schema but it still requires relate schema
![image](https://github.com/prisma/prisma/assets/101233364/ee174058-483c-4bbf-8692-a070d7a05bd4)
![image](https://github.com/prisma/prisma/assets/101233364/de30ee24-e785-4e88-ac70-539d1201eef6)

**Reply by janpio** [reply]
There is currently not way to isolate Introspection to one schema if that schema includes references to another schema.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
